base_layer/transaction_components/src/key_manager/inner.rs

```diff
@@ -1592,6 +1683,24 @@ where TBackend: TransactionKeyManagerBackend + 'static
         Ok(CompressedPublicKey::new_from_pk(public_key))
     }
 
+    pub async fn stealth_address_script_spending_key_id(
```

Comment:
you dont need this metod, 
this key here is wrong and its not going to work correctly
all you need to do if you want the `stealth_address_script_spending_key_id` is this:
```rust
let script_key = TariKeyId::Derived {
                                key: SerializedKeyString::from(commitment_mask_key_id.to_string()),
                            };
```
look here: https://github.com/tari-project/tari/blob/b0cfe304d6b5eeceb631cb9979f8d223b4520516/base_layer/wallet/src/output_manager_service/service.rs#L2907